docs/source/en/perf_train_cpu.mdx

```diff
@@ -58,3 +58,40 @@ Take an example of the use cases on [Transformers question-answering](https://gi
 <b>--use_ipex \</b>
 <b>--bf16 --no_cuda</b></pre> 
 
+### Performance Examples
+
+BERT-base, DistilBERT-base and BERT-large are selected as examples with different model sizes to demonstrate the BFloat16 training performance achieved by IPEX and speed-up ratio against Stock PyTorch FP32 on the 3rd Generation Xeon® Scalable Processors (Intel® Xeon® Platinum 8380H CPU). All the benchmarkings are using [IPEX CPU launcher](https://intel.github.io/intel-extension-for-pytorch/cpu/latest/tutorials/performance_tuning/launch_script.html) to set system configs to ensure a best-to-best comparison, like numactl (--node_id 0 , i.e., one socket full cores) and jemalloc (--enable_jemalloc).
```

Comment:
Like the checkpoint names to their model Hub page on hf.co

Reply:
Have refined the name with the checkpoint names and their link to the model Hub page on hf.co